### PR TITLE
LPS-89497 configuration upgrade

### PR DESCRIPTION
**Pull request by dantewang**

### CODE REVIEW

**Comment by dantewang** [comment]
ci:test:sf

**Comment by dantewang** [comment]
ci:test:relevant

File: modules/apps/static/portal-configuration/portal-configuration-persistence-impl/src/main/java/com/liferay/portal/configuration/persistence/internal/upgrade/ConfigurationUpgradeStepFactoryImpl.java [hunk 1]
```diff
@@ -59,6 +61,62 @@ public UpgradeStep createUpgradeStep(String oldPid, String newPid) {
...
+
+						String oldServicePid = dictionary.get("service.pid");
+
+						String newServicePid = StringUtil.replace(
+							oldServicePid, oldPid, newPid);
+
+						dictionary.put("service.pid", newServicePid);
+
+						String oldUri = dictionary.get(
+							"felix.fileinstall.filename");
```
Comment (tinatian) [inline modules/apps/static/portal-configuration/portal-configuration-persistence-impl/src/main/java/com/liferay/portal/configuration/persistence/internal/upgrade/ConfigurationUpgradeStepFactoryImpl.java]: @dantewang @yuhai It is possible that factory configuration does not have this property "felix.fileinstall.filename" set. 
Also it is possible that non-factory config has this set, it should also be updated to the new file path.

Please fix and resend.

Thanks.
Tina

**Comment by tinatian** [comment]
Please see comment above.

Tina.

**Pull request closed without merging**